packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -57,11 +57,31 @@ export const ConnectedRichTextEditor = (props: ConnectedProps) => {
 
   const plugins = React.useMemo(() => getPlugins(props.sdk, tracking), [props.sdk, tracking]);
 
+  React.useEffect(() => {
+    if (!editor) {
+      return;
+    }
+
+    const docFromAdapter = toSlatejsDocument({
+      document: props.value || Contentful.EMPTY_DOCUMENT,
+      schema,
+    });
+    const doc = sanitizeIncomingSlateDoc(docFromAdapter);
+
+    // Slate throws an error if the value on the initial render is invalid
+    // so we directly set the value on the editor in order
+    // to be able to trigger normalization on the initial value before rendering
+    editor.children = doc;
+    Editor.normalize(editor, { force: true });
+    // We set the value so that the rendering can take over from here
+    setValue(editor.children);
+  }, [props.value, editor]);
+
   return (
     <div className={styles.root} data-test-id="rich-text-editor">
       <Plate
         id={getContentfulEditorId(props.sdk)}
-        initialValue={value}
```

Comment:
Curious, what is the difference between using `initialValue` or `value` in this case?

Reply:
Probably I just let it over there after trying different things. It does not seem to make a difference.
Anyway I'd use it now that the first value we pass is an empty document and we change it in the effect, they call it [controlled value](https://plate.udecode.io/docs/Plate#value)